LOAN/Loans/LoansParties.rdf

```diff
@@ -69,79 +93,34 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-loan-loan-pty;Cosigner">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-loan-pty;LoanBorrower"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-dbt;Borrower"/>
```

Comment:
Based on the note, it seems that a Cosigner is not a Borrrower.

Reply:
Actually, I asked Bruce and a co-signer is considered a co-borrower, and has to jump through the same hoops that the borrower does, in case the borrower can't meet their obligations.  The definitions in this ontology need work and it may be merged with others, but at least that is correct.